Tasks/JenkinsQueueJobV2/task.loc.json

```diff
@@ -101,6 +101,15 @@
         "resizable": "true",
         "rows": "4"
       }
+    },
+    {
+      "name": "strictDefinition",
```

Comment:
Could it make sense to make it more specific, like 'failOnUnstableResultStatus'?

Reply:
@kuleshovilya Probably should also update the label if you decide to change this

Reply:
Yeah, something like that would sound better, although I think we can shorten it a bit to 'failOnUnstableResult', status is implied anyways.

Reply:
@kuleshovilya yes, make sense 👍